app/build.gradle

```diff
@@ -173,7 +173,7 @@ dependencies {
     implementation 'com.lsjwzh:materialloadingprogressbar:0.5.8-RELEASE'
 
     // Customizable TextView
-    implementation 'com.libRG:customtextview:2.4'
+    implementation files("/Modules/customtextview-2.1.aar");
```

Comment:
I think you have an extra / at the beginning. And a semicolon at the end isn't needed
Should just be `files("Modules/customtextview-2.1.aar")`

Reply:
Made the changes to reflect it. Just a style preference I have from work.